peering/management/commands/configure_routers.py

```diff
@@ -25,8 +26,36 @@ def add_arguments(self, parser):
             action="store_true",
             help="Delegate router configuration to Redis worker process.",
         )
+        parser.add_argument(
+            "-c",
+            "--config",
+            type=str,
+            help="Override default configuration to use for config generation/deployment. Give the name of the configuration.",
+        )
+
+    def process(
+        self,
+        router,
+        quiet=False,
+        as_task=False,
+        no_commit_check=False,
+        config_override=False,
```

Comment:
If `config_override` is supposed to be a string containing the name of the config, the default value would probably better as `""` or `None`.